apps/src/templates/manageStudents/PasswordReset.jsx

```diff
@@ -40,39 +49,66 @@ class PasswordReset extends Component {
       }
     };
 
-    $.ajax({
-      url: `/dashboardapi/sections/${sectionId}/students/${studentId}`,
+    fetch(`/dashboardapi/sections/${sectionId}/students/${studentId}`, {
       method: 'PATCH',
-      contentType: 'application/json;charset=UTF-8',
-      data: JSON.stringify(dataToUpdate)
+      headers: {
+        'Content-Type': 'application/json;charset=UTF-8'
+      },
+      body: JSON.stringify(dataToUpdate),
+      credentials: 'include'
```

Comment:
Mostly curious, why 'include' instead of 'same-origin' here?

Reply:
Oh I was switching that while trying to debug, I'll change it back to 'same-origin'